### PR TITLE
Feature flex refactor

### PR DESCRIPTION
**Pull request by susrisha**
- Flex is refactored with new data
- Seeder used in some cases (for upload)
Reference Task : [Task 322](https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/322): Flex API tests

### CODE REVIEW

**Review by priya-badmanji — APPROVED** [review]

File: src/__tests__/gtfsflex.test.ts [hunk 1]
```diff
@@ -20,66 +35,202 @@ describe('GTFS FLEX API', () => {
...
-        );
-
-        expect(uploadedFileResponse.data != "").toBe(true);
-
-    }, 50000);
-});
\ No newline at end of file
+        })
+
+        it('When passed with valid token and tdei_service_id, shold return records of same service', async ()=>{
```
Comment (ayydevara) [inline src/__tests__/gtfsflex.test.ts:71]: nit: typo `shold`

File: src/__tests__/gtfsflex.test.ts [hunk 2]
```diff
@@ -20,66 +35,202 @@ describe('GTFS FLEX API', () => {
...
+                expect(element.tdei_service_id).toEqual(tdei_service_id);
+            });
+        })
+
+        it('When passed with valid token and valid recordId, should return 200 status with only single record with same record Id', async () => {
+           
+            let tdei_record_id = '2bf7f70127b146cbb96319b5d39ada93';
+            let flexApi = new GTFSFlexApi(configuration);
+
+            const files = await flexApi.listFlexFiles(undefined,undefined,undefined,undefined,undefined,tdei_record_id);
```
Comment (ayydevara) [inline src/__tests__/gtfsflex.test.ts:90]: Probably OK in this scenario. 

But in general, if this is intentional we should pass `null`(usually explicit). `undefined` (usually implicit) generally means a value is defined somewhere, but it lost its value somewhere in the code flow.

File: src/__tests__/gtfsflex.test.ts (hunk 1 above)
Comment (susrisha) [reply src/__tests__/gtfsflex.test.ts:71]: will correct

File: src/__tests__/gtfsflex.test.ts (hunk 2 above)
Comment (susrisha) [reply src/__tests__/gtfsflex.test.ts:90]: the function doesnt allow passing `null` in this parameters. Probably the generated code issue

**Comment by ayydevara** [comment]
Please delete stale branches after PR merge.

**Pull request merged**